docs/csharp/misc/cs3001.md

```diff
@@ -10,7 +10,7 @@ ms.assetid: c4f3e247-5e47-4182-b415-c573fb1a152f
 # Compiler Warning (level 1) CS3001
 Argument type 'type' is not CLS-compliant  
   
- A [public](../../csharp/language-reference/keywords/public.md), [protected](../../csharp/language-reference/keywords/protected.md), or `protected internal` method must accept a parameter whose type is compliant with the Common Language Specification (CLS). For more information on CLS Compliance, see [Writing CLS-Compliant Code](http://msdn.microsoft.com/library/4c705105-69a2-4e5e-b24e-0633bc32c7f3) and [Language Independence and Language-Independent Components](../../../docs/standard/language-independence-and-language-independent-components.md).  
+ A [public](../../csharp/language-reference/keywords/public.md), [protected](../../csharp/language-reference/keywords/protected.md), or `protected internal` method must accept a parameter whose type is compliant with the Common Language Specification (CLS). For more information on CLS Compliance, see [Writing CLS-Compliant Code](https://msdn.microsoft.com/library/4c705105-69a2-4e5e-b24e-0633bc32c7f3) and [Language Independence and Language-Independent Components](../../../docs/standard/language-independence-and-language-independent-components.md).  
```

Comment:
nit: Writing CLS-Compliant code can be deleted. All of its content is contained in the Language Independence topic.